test/functional/wallet_assumeutxo.py

```diff
@@ -57,6 +59,7 @@ def run_test(self):
         """
```

Comment:
Should the docs be updated to mention the added node?

Reply:
missed this, it could probably be done in a follow-up such as #30455